list/LIST_HEADING_README.md

```diff
@@ -0,0 +1,162 @@
+# List (Heading) Mapping
+
+## XML HL7 > JSON FHIR
+
+A List (Heading) is created from a [List (Topic)](./LIST_TOPIC_README.md) member (referred to as `Topic`) and a `CompoundStatement` within the same `ehrComposition` which has a classCode of `CATEGORY`
```

Comment:
Do we want to make it clear that the CompoundStatement referenced here is a descendent of the related Topics CompoundStatement?

Reply:
Added clarification as suggested.